example/lib/main.dart

```diff
@@ -23,10 +23,15 @@ class _MyGaugeExampleState extends State<MyGaugeExample> {
             SizedBox(
               child: LinearGauge(
                 value: 50.0,
+                primaryRulersHeight: 25,
```

Comment:
Please remove the unwanted properties which dosent effect the feature

Reply:
removed